web/src/beta/features/Visualizer/Crust/Widgets/Widget/builtin/GoogleMapSearch/index.tsx

```
@@ -1,34 +1,394 @@
+
+const removeGoogleMapsScript = () => {
+  const scripts = document.querySelectorAll<HTMLScriptElement>(
+    'script[src*="maps.googleapis.com/maps/api/js"]'
+  );
+  scripts.forEach((script) => {
+    script.remove();
+  });
+  if (window.google) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    delete window.google;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)